Source: @pullmoll
Upstream: no
Reason: ffitarget.h no longer defines FFI_SYSV for *nix

--- src/c/ffi.d	2016-12-19 11:25:00.000000000 +0100
+++ src/c/ffi.d	2020-04-25 13:29:05.385149890 +0200
@@ -133,8 +133,6 @@
 #elif defined(X86_WIN64)
   {@':win64', FFI_WIN64},
-#elif defined(X86_ANY) || defined(X86) || defined(X86_64)
-  {@':cdecl', FFI_SYSV},
-  {@':sysv', FFI_SYSV},
+#elif defined(X86_64)
   {@':unix64', FFI_UNIX64},
 #endif
 };
